benchmarks/src/jmh/java/reactor/AssemblyTraceBenchmark.java

```diff
@@ -44,25 +45,33 @@
 
 	static final String JAVA_8 = "/Library/Java/JavaVirtualMachines/jdk1.8.0_192.jdk/Contents/Home/bin/java";
 
-	static final String JAVA_11 = "/Library/Java/JavaVirtualMachines/openjdk-11.0.1.jdk/Contents/Home/bin/java";
+	static final String JAVA_11 = "/Library/Java/JavaVirtualMachines/openjdk-11.0.2.jdk/Contents/Home/bin/java";
 
 	@Param({"10", "40", "80"})
 	int stackSize;
 
 	@Benchmark
 	@Fork(
-			jvm = JAVA_8,
 			jvmArgsAppend = {"-XX:-OmitStackTraceInFastThrow", "-Dreactor.trace.operatorStacktrace=true"}
 	)
-	public void withTracingOnJDK8(Blackhole bh) {
+	public void withTracing(Blackhole bh) {
 		stack(stackSize, bh);
 	}
 
-	@Benchmark
-	@Fork(
-			jvm = JAVA_11,
-			jvmArgsAppend = {"-XX:-OmitStackTraceInFastThrow", "-Dreactor.trace.operatorStacktrace=true"}
-	)
+//	@Benchmark
+//	@Fork(
+//			jvm = JAVA_8,
+//			jvmArgsAppend = {"-XX:-OmitStackTraceInFastThrow", "-Dreactor.trace.operatorStacktrace=true"}
+//	)
+//	public void withTracingOnJDK8(Blackhole bh) {
+//		stack(stackSize, bh);
+//	}
+
+//	@Benchmark
+//	@Fork(
+//			jvm = JAVA_11,
+//			jvmArgsAppend = {"-XX:-OmitStackTraceInFastThrow", "-Dreactor.trace.operatorStacktrace=true"}
+//	)
 	public void withTracingOnJDK11(Blackhole bh) {
```

Comment:
can you comment on why removing JDK11 bench in this PR? is the rationale that the bench would be run via a manually triggered CI job on both JDKs? cc @bsideup any problem you foresee here?

in any case, it doesn't seem to serve a purpose to keep this (either leave the whole bench untouched, or remove this dead code)

Reply:
its not removed, the same code just named withTracing and withoutTracing is still here. I just should have removed those commented ones